Docker/Container.php

```diff
@@ -186,20 +202,16 @@ private function runWithoutLogger(Process $process)
 
     private function runWithLogger(Process $process, $containerName)
     {
-        $server = ServerFactory::createServer($this->getImage()->getLoggerServerType());
+        $server = ServerFactory::createServer($this->getImage()->getSourceComponent()->getLoggerServerType());
         /* the port range is rather arbitrary, it intentionally excludes the default port (12201)
             to avoid mis-configured clients. */
         $containerId = '';
         $server->start(
             12202,
```

Comment:
@ujovlado  no, ale ty porty nepotrebujes mapovat, protoze je otevira host (kterej sice bezi v containeru, ale nejde to z nej ven)